menderqa_pipeline.go

```diff
@@ -308,7 +308,14 @@ func getBuildParameters(log *logrus.Entry, conf *config, build *buildOptions, pr
 
 	// set the correct integration branches if we aren't performing a pull request against integration
 	if build.repo != "integration" && build.repo != "meta-mender" {
-		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("integration"), Value: build.baseBranch})
+		revision := build.baseBranch
+		if _, exists := prsRepos["integration"]; exists {
+			revision = prsRepos["integration"]
+		}
+		buildParameters = append(buildParameters,
```

Comment:
Can you add meta-mender to the test also, to see if this needs special handling also? I suspect it's needed :thinking:

Reply:
adding. I have no idea. Let's see what happens.

Reply:
ok, since with 8c3ee636e82c38295db2f8f5b321b5014c3ae0d4 I added only --pr to the test and it passed, then it needs special handling.

Reply:
:tada: Saved us a fix later on

Reply:
good catch! fixed.